packages/common/core-interfaces/src/exposedInternalUtilityTypes.ts

```diff
@@ -903,7 +903,7 @@ export namespace InternalUtilityTypes {
 			Controls extends FilterControlsWithSubstitution
 			? /* test for 'any' */ boolean extends (T extends never ? true : false)
 				? /* 'any' => */ Controls["DegenerateSubstitute"]
-				: Options["IgnoreInaccessibleMembers"] extends "ignore-inaccessible-members"
+				: Options extends { IgnoreInaccessibleMembers: "ignore-inaccessible-members" }
```

Comment:
Its unclear how somone is supposed to find the docs on JsonSerializableOptions: IgnoreInaccessibleMembers from here.

Up on line 871 I think you should replace

```
{
			IgnoreInaccessibleMembers?: "ignore-inaccessible-members";
		}
```

with `Pick<JsonSerializableOptions, "IgnoreInaccessibleMembers">`

so its clear those are the same field, and that documentation applies.

THen here you could do:

```suggestion
				: Options extends Required<Pick<JsonSerializableOptions, "IgnoreInaccessibleMembers">>
```

Reply:
That would be a cyclic file import unless that type is relocated, which didn't feel the best for readability for the API file. I am trying to keep the exposed options a bit separated from the internals. See also internal `FilterControls` that is like part of the `JsonSerializableOptions` -- I don't want to have those interdependent.
But, yes, there are no docs now in this place for that internal option. I will add a comment.